src/main/java/bio/terra/workspace/service/buffer/BufferService.java

```diff
@@ -1,17 +1,24 @@
 package bio.terra.workspace.service.buffer;
 
 import bio.terra.buffer.api.BufferApi;
+import bio.terra.buffer.api.UnauthenticatedApi;
 import bio.terra.buffer.client.ApiClient;
 import bio.terra.buffer.client.ApiException;
 import bio.terra.buffer.model.HandoutRequestBody;
 import bio.terra.buffer.model.PoolInfo;
 import bio.terra.buffer.model.ResourceInfo;
+import bio.terra.buffer.model.SystemStatus;
 import bio.terra.workspace.app.configuration.external.BufferServiceConfiguration;
 import bio.terra.workspace.app.configuration.spring.TraceInterceptorConfig;
+import bio.terra.workspace.generated.model.SystemStatusSystems;
```

Comment:
Is there a better name for this? The added benefit is avoiding a collision with the buffer one.

Reply:
I filed a bug on the backlog to move this to the common library... not sure I can/should rename in the interim